CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## 3.8.1
```

Comment:
Thanks; i prefer to write the changelog entries myself based on the commit titles and PR titles :-)